draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -225,7 +227,13 @@ when, and only when, they appear in all capitals, as shown here.
 
 This document additionally uses the TLS presentation language defined in {{Section 3 of !RFC8446}}.
 
-# Overview
+## Time {#time}
+
+All time computations in this document are represented by POSIX timestamps, defined in this document to be integers containing a number of seconds since the Epoch, defined in Section 4.16 of {{POSIX}}. That is, the number of seconds after 1970-01-01 00:00:00 UTC, excluding leap seconds. A UTC time is converted to a POSIX timestamp as described in {{POSIX}}.
```

Comment:
Positive integers I presume.

Reply:
What, you don't want to get a time machine and start a CA in the 1800s? :-)

I suspect this should go the same way as the question below. If we want to be able to say "10 seconds before some time", we should probably allow negative numbers just so the operation is defined everywhere. If not, probably doesn't matter either way.

Reply:
(Resolving to negatives allowed per other thread. It'll never actually happen but keeps the operations complete.)